src/net/SCTP/Association.cs

```diff
@@ -37,6 +37,7 @@ namespace SIPSorcery.Net.Sctp
 {
     public abstract class Association
     {
+        protected object myLock = new object();

```

Comment:
I don't see this used anywhere? Is it needed?

Reply:
No it isn't. I was planning on another pull request to change instances of lock(this) since it isnt recommend.

https://stackoverflow.com/questions/11788582/best-practices-in-using-a-lock